<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Elija una opci贸n:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="opcion1">
      <property name="text">
       <string>Opci贸n 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="opcion2">
      <property name="text">
       <string>Opci贸n 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="opcion3">
      <property name="text">
       <string>Opci贸n 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mensaje">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="aceptar">
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
